packages/firestore/src/core/query.ts

```diff
@@ -28,7 +28,12 @@ import { FieldPath, ResourcePath } from '../model/path';
 import { assert, fail } from '../util/assert';
 import { Code, FirestoreError } from '../util/error';
 import { isNullOrUndefined } from '../util/types';
+import { Target } from './target';
 
+/**
+ * Represents a local query, it can be run against local storage, as well as be converted
```

Comment:
Nit: s/it can/which can/

"Local storage" is also a very ambiguous term on Web. I believe you are referring to the Firestore concept (LocalStore) here rather than Web Storage.